edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -1,12 +1,21 @@
 {% load i18n %}
 {% load ace %}
+{% load static %}
 
 {% get_current_language as LANGUAGE_CODE %}
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 
 {# This is preview text that is visible in the inbox view of many email clients but not visible in the actual #}
 {# email itself. #}
 
+<style type="text/css">
+
+    .logo-image {
+        width: 225px;
```

Comment:
Why 225px? Is there an special requirement? How does it look like in mobile views? @andrey-canon

Reply:
That is the value that was in the inline styles

Reply:
How does it look like in mobile views? @andrey-canon

Reply:
good